lib/CreepPlayers/NecromancerHealPlayer.js

```diff
@@ -0,0 +1,23 @@
+
+var BaseCreepPlayer = require('./BaseCreepPlayer');
+var Spawn = require('./Behaviors/Spawn');
+var Run = require('./Behaviors/RunDumb');
+var KiteWithFriend = require('./Behaviors/KiteWithFriendDumb');
+var Heal = require('./Behaviors/Heal');
+
+
+function NecromancerPlayer(creepController) {
+    this.cc = creepController;
+
+    var spawn = new Spawn(this.cc);
+    var run = new Run(this.cc);
+    var heal = new Heal(this.cc);
+    var kite = new KiteWithFriend(this.cc);
+
+    this.behaviors = [heal, spawn, kite, run];
```

Comment:
heal is going to take precedence over everything which means as soon as you attack them, they will just sit there and heal. Is that correct?